### DISCUSSION TITLE
再见二丁目 | /cn/2022/06/2nd-street/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1589] [Comments]
当人群的脚步　川流不息时
当满街的桦树　随风舞叶时
唱片店内　突然传来一首
我听过无数遍的
再见二丁目
[&hellip;] 唔&hellip;&hellip;大约每个人心中
都有一处　遥远的二丁目
在那里
岁月的脚步　不肯驻足
飞舞的树叶　永远飞舞
[&hellip;] 不知何故
过去我　一直以为
再见的意思　是再相见
而现在　才突然醒悟
再见　只是告别与不见
[&hellip;] &hellip;

https://yihui.org/cn/2022/06/2nd-street/

**Comment by Yemiao1** [comment]
再见二丁目，我喜欢的歌。在欧洲留学期间有段时间一直看你的博客，回国几年了，今天又找到你的网站，哈哈，来看看。

**Reply by yihui** [reply]
《再见二丁目》是我最喜欢的粤语歌。

谢谢你的关注！之前留学是在维也纳吗？

**Reply by Yemiao1** [reply]
哈哈，品味高尚，这首歌我也极其喜欢，曾经最喜欢的歌，单曲循环不下200次。你记性真好，我是在维也纳留学的，哈哈，回国后找工作几经周折，都没有关注你了。今天想起来，搜到了你的网站。

**Reply by yihui** [reply]
好记性不如烂键盘。其实我并不记得你五年前来过，但我用三个关键词放狗一搜就找到了你的个人页面，只是不能百分百确定那是你。现在确认了你是在维也纳留学，那我找的页面就是对的了。然后我回头在自己的留言区搜了一下，才找到 2019 年的留言：https://github.com/yihui/yihui.org/discussions/287#discussioncomment-4417777 我记得那条羽毛球勾对角的留言，不过此前我已经完全忘了那是你留的，现在又重新对上了号。

我不知道你留学期间有没有遇到过 Fritz Leisch，他是 R 核心团队成员（虽然早就不活跃了），[几个月前不幸因病去世](https://www.r-project.org/doc/obit/fritz.html)。

**Reply by Yemiao1** [reply]
厉害了，哈哈 我本来写“在奥地利留学”，又改成了欧洲，没想到你还是能知道，佩服。我也记得给你评论过关于羽毛球，但是没找到。刚顺着你的#287点进去，然后看到了我5年前的留言。然后又发现你是我们湖北人。
我其实不会R语言，一直想学，方便我们作图。所以也没有有幸认识Fritz Leisch，很可惜。

**Reply by Yemiao1** [reply]
刚搜了下他，发现是我们BOKU计算机学院的院长，真的很可惜。

**Reply by yihui** [reply]
是的，英年早逝。要不是他二十多年前在 R 里发明了 Sweave，我后来肯定打不出 knitr/R Markdown 的江山，然后现在还不知在哪里默默搬砖呢。

话说你这名字听起来简直是生来就是农学博士的命。来，老夫便以“叶”字开头，草草[拼凑](https://github.com/yihui/yihui.org/discussions/208#discussioncomment-4436409)两句：

> 一叶浮萍归大海，五载老乡又相逢。  
> 再见何必二丁目，举杯共邀半月升。

**Reply by Yemiao1** [reply]
谢谢大佬，竟然知道了我的姓（名），经你提醒我也想起来了你是我老乡，我是湖北麻城的。你太厉害了，为你骄傲。谢谢你的诗文。

**Reply by Yemiao1** [reply]
也为认识你骄傲。上次跟我师兄吹起来 我说我认识你。我师兄也会R语言，他说刚开始学的时候也是跟你学的。

**Comment by Yemiao1** [comment]
还可以给你推荐一些歌，《月球下的人》、《倾城》、《捞月亮的人》。（我真是安利小能手，哈哈）
